pkg/project/auth/reviewer.go

```diff
@@ -0,0 +1,66 @@
+package auth
+
+import (
+	"fmt"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
+	"github.com/openshift/origin/pkg/client"
+)
+
+// Review is a list of users and groups that can access a resource
+type Review interface {
```

Comment:
The scenario here seems like a common pattern we have for abstracting fetching a response by an identifier, except here we translate the response (and in other cases we return our API objects).